src/main/java/com/microsoft/azure/functions/annotation/KafkaTrigger.java

```diff
@@ -24,7 +24,7 @@
  * <ul>
  *     <li>Any native Java types such as int, String, byte[]</li>
  *     <li>Nullable values using Optional&lt;T&gt;</li>
- *     <li>Any POJO type</li>
+ *     <li>Any POJO type for Cardinality.One</li>
```

Comment:
What about Any POJO type, currently supported only for Cardinality.One

Reply:
Very good one! Thanks!